fission-cli/library/Fission/CLI/Types.hs

```diff
@@ -606,11 +606,11 @@ instance
 
         Turtle.export "IPFS_PATH" $ Text.pack ipfsRepo
 
-        process <- startProcess . fromString $ intercalate " "
+        process <- startProcess . fromString $ unwords
```

Comment:
`unwords = intercalate " "`

This was a suggestion from HLint

Reply:
"unwords". naming is hard.

Reply:
@walkah haha yeah. Some context here: `words` is a function that breaks a string by spaces into a list of words. This is the reverse operation.